manage.py

```diff
@@ -1,12 +1,12 @@
 #!/usr/bin/env python
 import os
+# Do this before *anything*.
+os.environ.setdefault('DJANGO_SETTINGS_MODULE', 'fjord.settings')
```

Comment:
Er, I'm 95% sure it fixes the issue. I'm not positive. For that, we'll have to test in production!